ramp-engine/ramp_engine/aws/api.py

```diff
@@ -57,7 +57,7 @@
 
 # how long to wait for connections
 WAIT_MINUTES = 2
-MAX_TRIES_TO_CONNECT = 5
+MAX_TRIES_TO_CONNECT = 1
```

Comment:
If we change this to 1, do we still need the `if n_try < max_tries_to_connect:` ? I guess leaving it gives us the option to increase `MAX_TRIES_TO_CONNECT` in future ?

Reply:
Yes. I prefer to leave it as a param for two reasons:
1. it is easier to test because for the test we can set this and the `WAIT_MINUTES` to low values so that the tests is done within reasonable time
2. we can easily update it in the future

Reply:
> it is easier to test because for the test we can set this and the WAIT_MINUTES to low values so that the tests is done within reasonable time

you mean `MAX_TRIES_TO_CONNECT` ? But yes if you want to use several tries in future, though adding it to queue might be better